translator/translate/logs/util/get_eks_cluster_name_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package util
+
+import (
+	"testing"
+)
+
+func TestBackoffSleep(t *testing.T) {
```

Comment:
I don't think we want a test that actually sleeps for a minute. Consider creating another function like `getBackoffDuration` and just verifying the values it gets.

```
github.com/aws/amazon-cloudwatch-agent/translator/translate/logs/util	66.231s	coverage: 1.3% of statements
```